views/pages/index.ejs

```diff
@@ -4,16 +4,18 @@
     <meta charset="utf-8">
     <title><%= Math.round(currently.temperature) %>° <%= currently.summary.charAt(0).toUpperCase() + currently.summary.slice(1).toLowerCase() %> <%= params.formatted_location %> Weather</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="Weather forecast that's high performance, open-source, collaborative, and accessible.">
     <meta property="og:email" content="julian@juliannorton.com">
     <meta name="robots" content="Index, follow">
     <meta http-equiv="refresh" content="600">
+    <meta name="theme-color" content="#639"/>
     <style><%- include('../../public/style/main.css') %></style>
     <link rel="manifest" href="/manifest.json">
     <link rel="apple-touch-icon" sizes="48x48" href="/images/default/favicon.ico">
     <link rel="mask-icon" href="/images/<%= currently.icon %>/safari-pinned-tab.svg?v=<%= currently.icon %>" color="#5bbad5">
     <link rel="shortcut icon" href="/images/<%= currently.icon %>/favicon.ico?v=<%= currently.icon %>">
+    <script src="/sw-registration.js"></script>
```

Comment:
Nit picking here, include a small EJS comment as to what `sw-registration.js` means or does or why it's important.

Reply:
Feel free to change the coment content. My english is not very good.